packs/data/abomination-vaults-bestiary.db/volluk-azrinae.json

```diff
@@ -3323,7 +3323,13 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "FastHealing",
+                        "type": "fast-healing",
```

Comment:
```suggestion
```
We should be able to leave the type out, since it defaults to fast healing instead of regeneration.

Reply:
It was added by the system itself, but sure, I can take it out.